config/sql/config.go

```diff
@@ -31,6 +31,9 @@ const (
 // ResourceConfigurators.
 func Configure(p *config.Provider) { //nolint:gocyclo
 	p.AddResourceConfigurator("google_sql_database_instance", func(r *config.Resource) {
+		// we need to workaround the newly added (with v4.56.0)
+		// of `Optional=True` in the native provider
+		config.MoveToStatus(r.TerraformResource, "instance_type")
```

Comment:
In `v4.56.0` of the Terraform Google provider, `Optional: true` has been added to the native schema of the `instance_type` attribute, which moves the generated field to `spec`. We need to move it back to `status`.